.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report
+about: Create a bug report for Clippy
+labels: L-bug
```

Comment:
Labels with emoji (like "L-bug :beetle:" ) seems to make issue template not work in my clone.